src/main/java/seedu/address/model/room/booking/exceptions/InactiveBookingCheckInException.java

```diff
@@ -0,0 +1,10 @@
+package seedu.address.model.room.booking.exceptions;
+
+/**
+ * Signals that there was an attempt to check-in an inactive booking.
+ */
+public class InactiveBookingCheckInException extends RuntimeException {
+    public InactiveBookingCheckInException() {
+        super("Cannot check-in a booking that is not active.");
```

Comment:
Does inactive mean that the booking start date is not today?

It would be nice to explain "inactive" to the user (which can be done in the user guide)

Reply:
Active means the booking period includes today. Thus, inactive means booking period does not include today. Note: **not** just start date.

Agree, will update later.